docker/ci.dockerfile

```diff
@@ -0,0 +1,10 @@
+ARG version
+FROM stitchem/stitchem-base-cuda${version}:latest
+
+ADD . stitchEm
+WORKDIR stitchEm
+RUN echo ${CUDA_SHORT} > cuda.version
+RUN mkdir build
+WORKDIR stitchEm
```

Comment:
`WORKDIR build` meant here?

Reply:
Indeed!

Reply:
this worked by accident, the mkdir is useless then :thinking: